### PR TITLE
feat(dashboard): add dashboard for portal service overview

### PR DESCRIPTION
**Pull request by Phil91**
## Description

Added a dashboard to display the following data for the portal services:
    - the last errors that occured
    - the amount of errors per statusCode
    - the warnings that occured
    - the avg elapsed time of the requests per service

## Why

To have a better overview of the errors and warnings that appear within the portal services

## Issue

N/A - Jira Issue: CPLP-1849

## Checklist

- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: apps/kube-prometheus-stack/dashboards/portal.json [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+  "templating": {
+    "list": []
+  },
+  "time": {
+    "from": "now-3h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Portal",
```
Comment (hzierer) [inline apps/kube-prometheus-stack/dashboards/portal.json:447]: Can you maybe use a less ambiguous name as it is visible for all teams. E.g. "Product Portal"

**Comment by hzierer** [comment]
Your dashboard contains hard-coded uids (like [these](https://github.com/catenax-ng/k8s-cluster-stack/blob/8723ea4c8684e8270314a6cfc68d4bcee2307bcb/apps/kube-prometheus-stack/dashboards/portal.json#L28)) which means it will only work with your grafana instance.
Can you please re-export the dashboard with the option "Export for sharing externally":

image

**Comment by Phil91** [comment]
> Your dashboard contains hard-coded uids (like [these](https://github.com/catenax-ng/k8s-cluster-stack/blob/8723ea4c8684e8270314a6cfc68d4bcee2307bcb/apps/kube-prometheus-stack/dashboards/portal.json#L28)) which means it will only work with your grafana instance. Can you please re-export the dashboard with the option "Export for sharing externally":
> 
> image

thanks for the explanaition, didn't knew that.

File: apps/kube-prometheus-stack/dashboards/portal.json (hunk 1 above)
Comment (Phil91) [reply apps/kube-prometheus-stack/dashboards/portal.json]: changed the naming

Comment (hzierer) [reply apps/kube-prometheus-stack/dashboards/portal.json]: It's still just "Portal" in the newly exported file... ;)

**Review by hzierer — COMMENTED** [review]
The import worked now, but I does not show any data. Shall we do a short video call to check what's wrong?

File: apps/kube-prometheus-stack/dashboards/portal.json (hunk 1 above)
Comment (Phil91) [reply apps/kube-prometheus-stack/dashboards/portal.json]: sorry, now it's correct

**Comment by Phil91** [comment]
> The import worked now, but I does not show any data. Shall we do a short video call to check what's wrong?

Thanks for the help, the latest changes are in. the queries should work now.

**Review by hzierer — APPROVED** [review]
import successful, LGTM

(small improvement for next version: a dashboard title would be nice)

**Comment by Phil91** [comment]
> import successful, LGTM
> 
> (small improvement for next version: a dashboard title would be nice)

I will remember to set the title in the next version

**Pull request merged**